### PR TITLE
all: migrate to gonum.org/v1/plot

### PR DESCRIPTION
**Pull request by sbinet**
Fixes gonum/plot#372.

### CODE REVIEW

File: vg/vgpdf/vgpdf.go [hunk 1]
```diff
@@ -15,7 +15,7 @@ import (
 	"math"
 
 	"bitbucket.org/zombiezen/gopdf/pdf"
-	"github.com/gonum/plot/vg"
+	"gonum.org/v1/plot/vg"
```
Comment (kortschak) [inline vg/vgpdf/vgpdf.go]: Please add a blank line above this.

File: vg/vgsvg/vgsvg.go [hunk 2]
```diff
@@ -18,7 +18,7 @@ import (
 	"math"
 
 	svgo "github.com/ajstarks/svgo"
-	"github.com/gonum/plot/vg"
+	"gonum.org/v1/plot/vg"
```
Comment (kortschak) [inline vg/vgsvg/vgsvg.go]: Here too.

File: vg/vgpdf/vgpdf.go (hunk 1 above)
Comment (sbinet) [inline vg/vgpdf/vgpdf.go]: done

File: vg/vgsvg/vgsvg.go (hunk 2 above)
Comment (sbinet) [inline vg/vgsvg/vgsvg.go]: done

**Comment by kortschak** [comment]
@sbinet Rebase this and pull the trigger?

**Comment by sbinet** [comment]
it's time.

**Pull request merged**